.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,22 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
```

Comment:
this was all auto generated 
except for `on: [push, pull_request]` We can dial this in where ever we like.